Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3286,7 +3286,7 @@ LazyIframeLoadingEnabled:
 
 LazyImageLoadingEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
We should move to shipping for fall release, or perhaps even sooner. I'm not opposed to removing the toggle for this.